src/wolfio.c

```diff
@@ -20,6 +20,7 @@
  */
 
 
+#define WOLFSSL_STRERROR_BUFFER_SIZE 1024
```

Comment:
Please make this overridable `#ifndef WOLFSSL_STRERROR_BUFFER_SIZE`

Reply:
have done, if we want it downstream compile time adjustable should it be moved to a header?